src/locales/en.json

```diff
@@ -82,6 +82,7 @@
   "amount_error_zero":"Amount has to be higher than 0.",
   "amount_error_funds":"Insufficient funds.",
   "balance_amount":"Your balance: {amount}.",
+  "by": "By",
```

Comment:
This expression should include username like this `By {username}`

Reply:
@bonustrack  ahh I see, updated to use `FormattedMessage` component with updated expression